### ISSUE TITLE
How to use loadAll function ?

### ISSUE THREAD

**Issue by Thomas-Fiolet** [tum-vision/LDSO#50]
Hello everyone,

I need to load a map from a .bin file. I have found the "loadAll" function in FullSystem and have a few questions about it :

- Is this function fully functional or is it just a basis for those who need to load information from a .bin file ?

- And in which way am I supposed to use it ? I tried to add it in the "run_dso_kitti.cc" just before the main for loop but it crashes at the execution. More precisely, it crashes when the addActiveFrame function is called, and even more precisely in the setPrecalcFunction function, when trying to resize the targetPrecalc vector.

Thx a lot !

**Comment by NikolausDemmel** [comment]
I personally haven't used it and it looks like it might not be up-to-date. I suspect @gaoxiang12 added it for LDSO development (I don't see it in DSO). It doesn't look like you can re-start the odometry. Maybe it was more an intermediate tool to test pose-graph optimization, or just to get the visualization for a saved run. Even for that I think it is not really complete at this stage, e.g. considering how it treats the poses:

```
        this->Tcw = SE3(Tcw);
        TcwOpti = Sim3(this->Tcw.matrix());
```

TLDR: Don't assume it is fit for any particular use case. You'd have to implement what you need yourself.
